src/plugins/inspector/public/ui/inspector_panel.scss

```diff
@@ -1,11 +1,15 @@
 .insInspectorPanel__flyoutBody {
-  // TODO: EUI to allow for custom classNames to inner elements
-  // Or supply this as default
-  > div {
+  .euiFlyoutBody__overflowContent {
```

Comment:
I'm going to create a separate PR - after you merge this one - to clean this up a bit. I've got some changes in the works, but they're going to take more time to iron out across the various implementations (i.e. I was close but realized it broke the dashboard layout ;) ).